### PR TITLE
MohammadH/ Homepage crowdin eu uk

### PR DESCRIPTION
**Pull request by mohammad-rework**
Changes:

-   Apply EU UK changes to Crowdin Homepage

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [x] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by habib-deriv — APPROVED** [review]

File: src/components/containers/visibility.tsx [hunk 1]
```diff
@@ -37,11 +38,14 @@ const domainBasedCheck = () => {
 
     useEffect(() => {
         if (window) {
-            const host_name = window.location.hostname
-            if (host_name.includes('eu')) {
+            const subdomain = window.location.hostname.split('.').slice(0, -2).join('.')
+            const eu_domains = ['eu', 'staging-eu']
+            const uk_domains = ['uk', 'staging-uk']
```
Comment (salarhfz-fs) [inline src/components/containers/visibility.tsx:43]: If there is a config file in the project to store constants like these, it's better to move these to the config file.

File: src/pages/home/_trade-types.tsx [hunk 2]
```diff
@@ -249,13 +272,15 @@ const TradeTypes = (): React.ReactNode => {
                 mb="40px"
                 tablet={{ mb: '16px' }}
             >
-                {localize('Trade the way you want with 3 exciting trade types')}
+                {is_row
+                    ? localize('Trade the way you want with 3 exciting trade types')
+                    : localize('Trade the way you want with 2 exciting trade types')}
```
Comment (yashim-deriv) [inline src/pages/home/_trade-types.tsx:277]:
```suggestion
                <Localize translate_text="Trade the way you want with {{trade_no}} exciting trade types" values={{ trade_no: is_row ? '3' : '2' }} />
```

There's a better way to handle dynamic text translations as I shown above. You can refer to the `General Implementation Guide` for additional info

File: src/pages/home/_trade-types.tsx [hunk 3]
```diff
@@ -249,13 +272,15 @@ const TradeTypes = (): React.ReactNode => {
...
             </Header>
 
             <DesktopWrapper>
                 <Flex>
                     <Carousel {...settings}>
-                        {ItemsDetails.map((item) => {
+                        {(is_row ? items_details_cr : items_details_eu_uk).map((item) => {
```
Comment (yashim-deriv) [inline src/pages/home/_trade-types.tsx:283]: There are repetition of this statement here and at line 295. Best to set it once and re-use it in both lines.

**Review by yashim-deriv — CHANGES_REQUESTED** [review]
Added some review comments

**Review by prince-deriv — APPROVED** [review]

File: src/pages/home/_our_platforms.tsx [hunk 4]
```diff
@@ -30,9 +33,13 @@ const OurPlatforms = () => {
...
-                        {localize(
-                            'Choose from 8 powerful platforms — each designed with your needs in mind',
-                        )}
+                        {is_row
+                            ? localize(
+                                  'Choose from 8 powerful platforms — each designed with your needs in mind',
+                              )
+                            : localize(
+                                  'Choose from 2 powerful platforms — each designed with your needs in mind',
+                              )}
```
Comment (yashim-deriv) [inline src/pages/home/_our_platforms.tsx:42]: Can refactor this block to use `<Localize` method as previous review.

**Review by yashim-deriv — APPROVED** [review]
A minor refactor. The rest looks good! 🙌🏼

**Review by salarhfz-fs — APPROVED** [review]

**Pull request merged**